src/Windows/curl_easy.h

```diff
@@ -40,7 +40,8 @@ class curl_easy
     static std::unique_ptr<curl_easy> create(

```

Comment:
Please add these changes to src/Linux/curl_easy.h too, otherwise it breaks the build

Reply:
thanks for pointing it out. Done